README.md

```diff
@@ -65,6 +65,21 @@ Code Overview
 
 These are the important parts of the Telemetry Server architecture.
 
+The simplified pipeline works as follows:
+
+1. Telemetry pings are generated by Firefox instances and are sent out from users' computers.
+    * To simulate this for testing purposes, run `python test/postGZ.py FOLDER_CONTAINING_PINGS` to submit all the pings in the specified folder.
+    * Sample pings can be found under one's Firefox profile, in the `saved-telemetry-pings` subfolder - they are human-readable JSON files.
+    * The `test/sample-saved_session-pings` folder can be used as an example: run `python test/postGZ.py test/test-saved_session-pings` while the Node server is running.
```

Comment:
Both `test/sample-saved_session-pings` and `test/postGZ.py` are missing from the PR.  Also, please use similar file naming to existing python code, so either "post_gz.py" or something even more descriptive like "submit_files.py"